editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1765,8 +1765,8 @@ void SpriteFramesEditor::_remove_sprite_node() {
 void SpriteFramesEditor::_fetch_sprite_node() {
 	Node *selected = nullptr;
 	EditorSelection *editor_selection = EditorNode::get_singleton()->get_editor_selection();
-	if (editor_selection->get_selected_node_list().size() == 1) {
-		selected = editor_selection->get_selected_node_list().front()->get();
+	if (editor_selection->get_top_selected_node_list().size() == 1) {
+		selected = editor_selection->get_top_selected_node_list().front()->get();
```

Comment:
Uh, unrelated, but it would be nice if this wasn't called here twice. Simply calling `get_top_selected_node_list()` will update the list for some reason.